src/console_cmds.cpp

```diff
@@ -614,6 +615,119 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	IniFile ini = { group_names };
+	ini.LoadFromDisk(path, NO_DIRECTORY);
+	const IniGroup *ban_group = ini.GetGroup("bans", false);
+
+	bool is_loaded = ban_group != nullptr;
+	if (!is_loaded) {
+		return -1;
+	}
```

Comment:
You don't even need the `is_loaded` variable then.
```suggestion
	if (ban_group == nullptr) return -1;
```